### PR TITLE
[MediaStreamTrack article] Update for launch

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Update for launch.

### CODE REVIEW

**Comment by tomayac** [comment]
@jpmedley Please merge this on time for the beta blog post.

**Comment by rachelandrew** [comment]
Hi @tomayac  - I just found this PR because @jpmedley asked me to write the MDN docs for the interfaces. The article mentions `writableControl` and `readableControl`, but these look to me to have been removed from the spec in https://github.com/w3c/mediacapture-transform/pull/41 

Want to make sure everything aligns with reality so if I'm entirely incorrect please let me know!

**Review by jpmedley — APPROVED** [review]

**Comment by tomayac** [comment]
> w3c/mediacapture-transform#41

Good catch, thanks for flagging this. I'll update the article accordingly.

**Comment by rachelandrew** [comment]
@tomayac Under https://deploy-preview-6034--web-dev-staging.netlify.app/mediastreamtrack-insertable-media-processing/#the-mediastreamtrackprocessor

It mentions there being two properties, and we now have a single item list.

Same for https://deploy-preview-6034--web-dev-staging.netlify.app/mediastreamtrack-insertable-media-processing/#the-mediastreamtrackgenerator

I guess those need to be "exposes a single property", and then just describe readable/writable in a paragraph rather than a list item.

**Comment by rachelandrew** [comment]
Also under https://deploy-preview-6034--web-dev-staging.netlify.app/mediastreamtrack-insertable-media-processing/#core-concepts both mention a control channel, which I think now isn't correct.

File: src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md [hunk 1]
```diff
@@ -111,42 +89,33 @@ components:
   [`VideoFrame`](https://w3c.github.io/webcodecs/#videoframe-interface) or
   [`AudioFrame`](https://w3c.github.io/webcodecs/#audioframe-interface)) objects. You can think of
   this as a track sink that is capable of exposing the unencoded frames from the track as a
-  `ReadableStream`. It also exposes a control channel for signals going in the opposite direction.
+  `ReadableStream`.
 - The `MediaStreamTrackGenerator`, which consumes a stream of media frames and exposes a
   `MediaStreamTrack` interface. It can be provided to any sink, just like a track from
   `getUserMedia()`. It takes media frames as input. In addition, it provides access to control
   signals that are generated by the sink.
```
Comment (tomayac) [inline src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md:96]:
```suggestion
  `getUserMedia()`. It takes media frames as input.
```

**Comment by tomayac** [comment]
Thanks for the eagle eye and looking at things holistically, @rachelandrew. PTyetAL :-)

**Review by rachelandrew — APPROVED** [review]
lgtm :)

**Pull request merged**